wrapper/src/main/java/software/amazon/jdbc/plugin/failover/FailoverConnectionPlugin.java

```diff
@@ -137,15 +138,19 @@ public class FailoverConnectionPlugin extends AbstractConnectionPlugin {
           "enableClusterAwareFailover", "true",
           "Enable/disable cluster-aware failover logic");
 
+  public static final AwsWrapperProperty FAILOVER_STRICT_READER =
+      new AwsWrapperProperty(
+          "failoverStrictReader", "false",
```

Comment:
Since the default value is false, can we rename this variable to `enableFailoverStrictReader`?

Reply:
And the description will be `Enable reader-only connection during failover`.

Reply:
nit: "Enable to enforce reader-only connection during reader failover" might be a bit clearer